app/src/main/java/com/app/foodforyou/view/DietListActivity.java

```diff
@@ -105,9 +104,9 @@ public void run() {
 
         adapter.setOnItemClickListener(new OnDietClickListener() {
             @Override
-            public void onDietItemClick(RecommendDietListAdapter.ViewHolder holder, View view, int position) {
-                Intent intent = new Intent(getApplicationContext(), RecommendDietFoodListActivity.class);
-                PreferenceManager.setString(mContext, "cntntsNo", String.valueOf(dataManager.getRecommendDietListResponses().get(position).getCntntsNo()));
+            public void onDietItemClick(DietListAdapter.ViewHolder holder, View view, int position) {
+                Intent intent = new Intent(getApplicationContext(), FoodListActivity.class);
+                PreferenceManager.setString(mContext, "cntntsNo", String.valueOf(dataManager.getDietListResponses().get(position).getCntntsNo()));
```

Comment:
cntntsNo should be static final, not hard coded

Reply:
Thanks! Which is better to make clean code?

1. Declare static final string variable in same class (ex. public static final String CONTENTS_NO)
2. Make to string value (ex. R.string.contentsNo)

Reply:
number 1 is recommended